templates/store/publisher-details.html

```diff
@@ -33,14 +33,15 @@ <h1 class="p-heading--two p-snap-heading__name">{{ publisher }}</h1>
             </span>
             {% endif %}
           </p>
+          <ul class="p-inline-list--middot u-float-left"></ul>
```

Comment:
Why do we need this empty `ul`?

Reply:
This is really to push things down as in the header of snap details page:

![image](https://user-images.githubusercontent.com/8167677/54680180-2045b300-4b01-11e9-8808-367da661d81b.png)

If there is text or not, this is always pushing things down so we can either have an empty UL or just fix this so the one pushing is the heading instead.

Reply:
I think it would be better to add a media-query for smaller screens to something within the header rather then empty elements?